File: 0_migration_2021/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for height
    int n;

    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);


    // Iterate from 1 and print number of hashes 

    for (int i = 1; i <= n; i++)
    {
        for (int k = (n - 1); k >= i; k--)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height:")
    if height > 0 and height < 9:
        break


for i in range(height - 1):
    print(".")

    for j in range(height):
        print("." * (height - 1), "#", end="")
        #print(, end="")
    print()
    height -= 1


